src/components/extensive/WorkdayCollapseGrid/types.ts

```diff
@@ -50,8 +50,8 @@ const CASTES: Dictionary<string> = {
 };
 
 const AGES: Dictionary<string> = {
-  youth: "Youth (15-24)",
-  adult: "Adult (24-64)",
+  youth: "Youth (15-29)",
+  adult: "Adult (29-64)",
```

Comment:
This change is affecting non-HBF values as well. I believe Claire is asking for the value to change for HBF only.